loadstone_front/src/app/menus/memory_map/mod.rs

```diff
@@ -133,7 +137,12 @@ fn add_internal_bank(
     internal_memory_map: &mut InternalMemoryMap,
     bank_start_address: u32,
     internal_flash: &FlashChip,
+    max_bank_count: usize,
 ) {
+    if internal_memory_map.banks.len() >= max_bank_count {
+        return;
```

Comment:
Hmm, not a fan of silent returns for unheld invariants. I would refactor this function to return `anyhow::Result`, even if the error case is discarded by the caller.